src/report/service/ErrorReportingService.ts

```diff
@@ -1,23 +1,28 @@
-import { Logger, LogLevel } from 'channelape-logger';
 import * as uuid from 'uuid';
+import { ChannelApeClient, Order } from 'channelape-sdk';
 
 import SqsMessageService from '../../aws/sqs/service/SqsMessageService';
 import ErrorReport from '../model/ErrorReport';
 import AwsCredentials from '../../aws/model/AwsCredentials';
+import ErrorReportModule from '../model/ErrorReportModule';
+import AdditionalFieldService from '../../channelape/service/AdditionalFieldService';
 
 export default class ErrorReportingService {
   private readonly sqsMessageService: SqsMessageService;
-  private readonly logger: Logger;
   private readonly channelApeWebAppDomainName: string;
+  private readonly channelApeClient?: ChannelApeClient;
+  private readonly businessId?: string;
 
   constructor(
     awsCredentials: AwsCredentials,
     awsErrorsQueueUrl: string,
-    logLevel: LogLevel | string,
-    stagingEnvironment = false
+    stagingEnvironment = false,
+    channelApeClient?: ChannelApeClient,
+    businessId?: string
```

Comment:
I fear that adding this to the constructor could cause issues on some of our external integrations. If a new instance of ErrorReportingService is not created every action, it could potentially be using the wrong businessId.

I think a cleaner way might be to add businessId as an optional param on ErrorReport

Do you think that would work for our EIs? If not we can live with this limitation. Just need to be aware of it.

cc: @csimko